videoframegrid.py

```diff
@@ -39,7 +39,7 @@ def _add_videoframe(self, videoframe):
 
     def _create_videoframe(self, vlc_instance, stream_url, stream_options, quality):
         """Creates a new LiveVideoFrame object."""
-        return LiveVideoFrame(vlc_instance, stream_url, stream_options, quality)
+        return LiveVideoFrame(self.parent, vlc_instance, stream_url, stream_options, quality)
```

Comment:
As `VideoFrameGrid` in turn is a `QGridLayout`, which is _not_ a `QWidget`, it cannot be used as a parent for other `QWidget` objects.